instrumentation-core/spring/build.gradle

```diff
@@ -0,0 +1,36 @@
+plugins {
+	id 'org.springframework.boot' version '2.3.0.RELEASE'
+	id 'io.spring.dependency-management' version '1.0.9.RELEASE'
+	id 'java'
+}
+
+group = 'io.opentelemetry.contrib'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '8'
+
+configurations {
+	compileOnly {
+		extendsFrom annotationProcessor
+	}
+}
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
```

Comment:
I'm sure things will get sorted out as the code gets in but one thought is that most dependencies should be `compileOnly` since spring boot starters actually enable features in a user's app. Instead, the general design should be recognizing when a user app is using a feature and instrumenting only those. `@ConditionalOnClass` tends to be very important for it :)

I guess this project will be similar to sleuth (maybe it could use it too? not sure). So will be a good reference point I think https://github.com/spring-cloud/spring-cloud-sleuth/tree/master/spring-cloud-sleuth-core